rename concept classifier and change serializationFormatVersion to 2023.1

enikao opened the pull request:
Only contains the JSON changes.

enikao committed:
renamed serialization "concept" -> "classifier"

enikao committed:
updated serializationFormatVersion to "2023.1"

ftomassetti commented:
Thank you @enikao for taking care of this! I think we may also need to update the dependencies to work with this new JSON format.  I think we should first update the dependencies, have all tests passing and only then merging this.

Also, I created a tag for the current version of the code, as this is the version that we refer to in the tutorial.

joswarmer committed:
Update to latest version:
Renaming:
 - LIonCore => LionCore
 - ConceptInterface => Interface
 Add empty `annotations: []`` (required) on each node
 Fixed three empty `key`s with with random values.

joswarmer committed:
Update language version (of M3) to 2023.1

joswarmer commented:
I updated the pricing.json further to make it up to date with the lastest LionWeb definitions.
